examples/examples-cpu/prefect/prefect-cloud-scheduled-scoring.ipynb

```diff
@@ -418,9 +418,9 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.6"
+   "version": "3.7.7"
   }
  },
  "nbformat": 4,
  "nbformat_minor": 4
-}
\ No newline at end of file
+}
```

Comment:
this is not something I did manually. I see these newline / no-newline changes frequently in diffs when I edit notebooks. No idea where they came from. I don't think they're problematic though.

Reply:
yeah I think my vs-code setup stripped those out when I was making changes here. I'll see if I can find a setting for that so it doesn't happen again.

Reply:
I think you should, for all types of files in all settings, not trim off the final trailing newline...

Reply:
Yep, agreed. This is the only file type where that happens, need to dig into the settings to find out why.